packages/devextreme/js/__internal/grids/grid_core/keyboard_navigation/m_keyboard_navigation.ts

```diff
@@ -160,6 +162,8 @@ export class KeyboardNavigationController extends modules.ViewController {
     this.focusedHandlerWithContext = this.focusedHandlerWithContext || this.focusedHandler.bind(this);
     this.renderCompletedWithContext = this.renderCompletedWithContext || this.renderCompleted.bind(this);
     this.rowsViewFocusHandlerContext = this.rowsViewFocusHandlerContext || this.rowsViewFocusHandler.bind(this);
+    // eslint-disable-next-line max-len
+    this.rowsViewFocusOutHandlerContext = this.rowsViewFocusOutHandlerContext ?? this.rowsViewFocusOutHandler.bind(this);
```

Comment:
Let's remove the ```eslint-disable-next-line``` comment and just write the ```??``` on the next line:
```suggestion
    this.rowsViewFocusOutHandlerContext = this.rowsViewFocusOutHandlerContext 
      ?? this.rowsViewFocusOutHandler.bind(this);
```